submissions/Ant-C-tech/html-css-popup/index.html

```diff
@@ -0,0 +1,162 @@
+                        </label>
+
+                        <div class="dropdown__moreElem">
+
+                            <a class="dropdown__item" href="#">
+                                <img class="dropdown__img" src="./img/chat.svg" alt="chat icon">
+                                <p class="dropdown__linkTitle">Chat</p>
+                            </a>
+
+                            <a class="dropdown__item" href="#">
+                                <img class="dropdown__img" src="./img/whiteboard.svg" alt="classroom icon">
+                                <p class="dropdown__linkTitle">Classroom</p>
+                            </a>
+
+                            <a class="dropdown__item" href="#">
+                                <img class="dropdown__img" src="./img/stamps.svg" alt="collection icon">
+                                <p class="dropdown__linkTitle">Collection</p>
+                            </a>
+
+                            <a class="dropdown__item" href="#">
+                                <img class="dropdown__img" src="./img/mars.svg" alt="mars icon">
+                                <p class="dropdown__linkTitle">Mars</p>
+                            </a>
+
+                            <a class="dropdown__item" href="#">
+                                <img class="dropdown__img" src="./img/writing.svg" alt="blogs icon">
+                                <p class="dropdown__linkTitle">Blogs</p>
+                            </a>
+
+                            <a class="dropdown__item" href="#">
+                                <img class="dropdown__img" src="./img/megaphone.svg" alt="promos icon">
+                                <p class="dropdown__linkTitle">Promos</p>
+                            </a>
+
+                            <p class="dropdown__footer">Even more from<a class="dropdown__footerLink"
+                                    href="https://kottans.org/" target="_blank"> Kottans</a> </p>
+
+                        </div>
+
+
+                    </div>
+                </div>
+
+                <a class="header__menuLink header__menuLink-bell" href="#">
+                    <img class="header__menuIcon header__menuIcon-bell" src="./img/bell.svg" alt="bell icon">
+                </a>
+                <a class="header__menuLink header__menuLink-last" href="#">
+                    <img class="header__menuIcon header__menuIcon-cat" src="./img/cat-in-black-silhouette.svg"
+                        alt="cat icon">
+                </a>
+            </div>
+        </div>
+        <img class="intro__img intro__imgAnimate" src="./img/cat-smile.jpg" alt="cat">
+        <p class="intro__footer">Icon by <a class="intro__footerLink"
+                href="https://www.flaticon.com/free-icon/maps_1673429?term=map&page=2&position=8&related_item_id=1673429"
+                target="_blank">Flaticon</a>
+        </p>
+    </div>
+
+</body>
+
+</html>
```

Comment:
could you please add an empty line at the end of the file?  If you see this symbol it means that an empty line is needed
![empty line](https://user-images.githubusercontent.com/10871447/99917473-39acf280-2d19-11eb-9090-c5d24be11a37.png)